### PR TITLE
Fix/new login stage 1

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/views/Auth/Universal/index.tsx [hunk 1]
```diff
@@ -79,82 +80,89 @@ const UniversalAuth = () => {
   }, [])
 
   return (
-    <section className={styles.wrapper}>
-      <section className={containerClasses}>
-        {currStep === 'select-login-method' && (
-          <>
-            <DynamicSelectAuthMethodForm
+    <>
+      <Head />
```
Comment (robertu7) [inline src/views/Auth/Universal/index.tsx:84]: better to have a title about authentication, please confirm with product team.

File: src/components/Dialogs/SetUserNameDialog/hook.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { useApolloClient } from '@apollo/react-hooks'
+import { toLower } from 'lodash'
```
Comment (robertu7) [inline src/components/Dialogs/SetUserNameDialog/hook.ts:2]: Use `lodash/toLower` for code splitting or just use `.toLowerCase()`

**Review by robertu7 — CHANGES_REQUESTED** [review]
please double-check the dialog spacing issues

File: src/views/Auth/Universal/index.tsx (hunk 1 above)
Comment (pitb2022) [reply src/views/Auth/Universal/index.tsx:84]: Confirm the use of the default title

**Review by robertu7 — APPROVED** [review]

**Pull request merged**